src/DynamoApplications/StartupUtils.cs

```diff
@@ -377,7 +377,7 @@ private static bool PreloadASM(string asmPath, out string geometryFactoryPath, o
                 HostAnalyticsInfo = info,
                 CLIMode = CLImode
             };
-            config.AuthProvider = CLImode ? null : new Core.IDSDKManager();
+            config.AuthProvider = CLImode || !string.IsNullOrEmpty(info.HostName) ? null : new Core.IDSDKManager();
```

Comment:
Host info can be passed by CLI or as param in out of process integrations so technically it may not be null all the time, I feel we will need more testing on this. Would you revert this for now, that is more conservative I think

Reply:
I'm not 100% sure what would be the best check here, maybe combine this with some other check, e.g. process name?